fontbe/src/glyphs.rs

```diff
@@ -258,14 +265,49 @@ impl Work<Context, Error> for GlyphWork {
                 };
                 context.set_glyph(name.clone(), base_glyph.clone().into());
 
-                (name, point_seqs_for_simple_glyph(ir_glyph, instances))
+                let mut contour_end = 0;
+                let mut contour_ends = Vec::with_capacity(base_glyph.contours().len());
+                for contour in base_glyph.contours() {
```

Comment:
maybe write-fonts' SimpleGlyph could have a method to get a vec of contour end-point indices

Reply:
How about if it comes up 1-2 more times we do that? - for now I'm not sure it will.